docs/integrators/deep-history-squad.md

```diff
@@ -0,0 +1,155 @@
+```
+
+:::note
+Currently, the API client has to perform the conversion from _desired timestamp_ to _block nonce_. In a future release, the API will directly support timestamp-based queries.
+:::
+
+> How much UTK were in the [`UTK / WEGLD` Liquidity Pool](https://explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgq0lzzvt2faev4upyf586tg38s84d7zsaj2jpsglugga) on [1st of October](https://explorer.elrond.com/blocks/cefd41e1e9bbe3ba023a695f412b99cecb15ef789475648ee7c31e7d9fef31d1)?
+
+```
+GET http://squad:8080/erd1qqqqqqqqqqqqqpgq0lzzvt2faev4upyf586tg38s84d7zsaj2jpsglugga/key/726573657276650000000a55544b2d326638306539?blockNonce=11410000
+```
+
+In the example above, the key `726573657276650000000a55544b2d326638306539` is decoded as `reserve\x00\x00\x00\nUTK-2f80e9`.
+
+## Public instance
+
+:::note
+As of October 2022, a public deep-history squad isn't yet available. The instance is being prepared and should be ready in November 2022.
+:::
+
+## On-premises instance
+
+Deep-history squads can be set up on-premises, just as regular observing squads. However, the storage requirements might increase over time - we'll tackle this [in a separate section](/integrators/deep-history-squad#handling-storage-requirements).
+
+Since each observer of a deep-history squad must have a non-pruned history, their non-regular databases have to be either **downloaded** or **reconstructed**, in advance.
+
+### Downloading non-pruned database
+
+:::note
+As of October 2022, a public repository with non-pruned databases for both _mainnet_ and _devnet_ is under construction. This repository would take the shape of a _Digital Ocean (S3-compatible) Space_. Once the repository is ready, the data can be downloaded via **[db-archive-scripts](https://github.com/ElrondNetwork/db-archive-scripts)** - documentation will follow.
+:::
+
+### Reconstructing non-pruned database
+
+An alternative to downloading a non-pruned history is to reconstruct it locally (on your own infrastructure). 
+
+Under the hood, the reconstruction process relies on the **[import-db](https://docs.elrond.com/validators/import-db/)** feature, which allows us to reprocess previosuly processed blocks - and, while doing so, for our purposes, we'll also retain the whole, non-pruned history. The `import-db` requires a **start  database** (placed in the folder `node-workdir/db`) and a **target database** (usually placed in the folder `node-workdir/import-db/db`). In the context of deep-history squads, _archives_ of such databases are called **time capsules**.
+
+It follows that, in order to reconstruct the history for an observer, we need (to download) **two time capsules**: a _start time capsule_ and a _target time capsule_. For reconstucting the history of a whole squad, `4 x 2` time capsules are required (to be downloaded).
+
+_Downloading_ the necessary time capsules and _unarchiving_ them is encapsulated in a step called **reconstruction bootstrapping**.
+
+#### Bootstrapping
```

Comment:
unfortunately, 4 `#`s won't make the item to be visible in the sidebar

Reply:
Indeed.